### PR TITLE
[bug]: fixed filters box functionality and default map view

### PR DESCRIPTION
**Pull request by nxanil**
@natabal @bmukheja 

Fixed the following errors:- 

1) [Remove sorting from label columns](https://github.com/nuagenetworks/visualization-framework/issues/763).

2) [Default view is not covering all the NSGs in Geo Map display when NSG's are placed across the world](https://github.com/nuagenetworks/visualization-framework/issues/765).

3) [Filter box functionality](https://github.com/nuagenetworks/visualization-framework/issues/762)

- Blank window with no search on "selection" search column in geomap/table - **Fixed**
 
- Delete button not working on the search filter box - Actually that was not a delete button, it is only for valid and invalid search string, so I have changed icons to get rid of confusion.  **[See 3rd](https://nhabuiduc.github.io/)** 
- When deleting with backspace, data shows up after a while - **I have make some changes to speed up response.**
- Some fields are not searchable at all even with correct values. There is no fixed rule when this happens or not - **not able to find the issue, will look by adding more complex data.**
- The tick mark, delete button and test are going out of the box - **fixed**
- The magnifier icon is still full size and appears ugly- **fixed**
- What is the use of magnifier icon anyways? - **removed**

### CODE REVIEW

File: SearchBar/AutoCompleteHandler.js [hunk 1]
```diff
@@ -9,4 +10,22 @@ export default class AutoCompleteHandler extends GridDataAutoCompleteHandler {
         var result = super.needOperators(parsedCategory);
         return this.scroll ? ["==", "!="] : result.concat(["startsWith"]);
     }
+
+    needValues(parsedCategory, parsedOperator) {
+        const found = _.find(this.options, f => f.columField == parsedCategory || f.columnText == parsedCategory);
```
Comment (natabal) [inline SearchBar/AutoCompleteHandler.js]: '==' should be '==='

File: SearchBar/AutoCompleteHandler.js [hunk 2]
```diff
@@ -9,4 +10,22 @@ export default class AutoCompleteHandler extends GridDataAutoCompleteHandler {
...
+        const parsedField = found ? found.columField : null;
```
Comment (natabal) [inline SearchBar/AutoCompleteHandler.js]: above this statement you should do 
```
if (found === null) {
     return []
}
```
And then remove the check on if (found != null)

File: SearchBar/AutoCompleteHandler.js [hunk 3]
```diff
@@ -9,4 +10,22 @@ export default class AutoCompleteHandler extends GridDataAutoCompleteHandler {
...
+
+        if (found != null && found.type == "selection" && this.data != null) {
```
Comment (natabal) [inline SearchBar/AutoCompleteHandler.js]: please use '===' and '!=='

**Comment by bmukheja** [comment]
1. There is 1 bug in this, when we clear the filter box, the table doesn't populate back

**Comment by nxanil** [comment]
> 1. There is 1 bug in this, when we clear the filter box, the table doesn't populate back for 3-4 seconds. Please make the duration lesser if possible to avoid confusion.

@bmukheja I have reduced setTimeout by 2/3rd and fix searching issue. Please have a look.

**Pull request merged**